client/src/component/candidateFlyout/candidateFlyout.js

```diff
@@ -60,6 +60,7 @@ const CandidateFlyout = ({ person, flyoutOpen, setFlyoutOpen }) => {
         {hasNominators &&
           nominations.map((nom) => (
             <WhiteContainer
+              key={nom.nominator}
```

Comment:
What is this for as well?

Reply:
react thing, every child in a list needs a unique key so it can do its magic